src/datepicker/examples.js

```diff
@@ -14,6 +14,7 @@ import {
 import {Popover, PLACEMENT} from '../popover/index.js';
 import {Input} from '../input/index.js';
 import tests from './examples-list.js';
+import {highlight} from 'lowlight';
```

Comment:
is this used?

```suggestion
```

Reply:
That's my editor being helpful by automatically adding imports